src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelector.tsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import ReactSelect, {Styles, ValueType} from 'react-select';
+
+import {Event} from 'app/types';
+
+import ThreadsSelectorOption from './threadsSelectorOption';
+import {Thread} from './getThreadDetails';
+
+const styles: Styles = {
+  control: provided => ({
+    ...provided,
+    width: '100%',
+    height: 28,
+    minHeight: 28,
+    border: '1px solid #c1b8ca',
+    borderColor: '#c1b8ca',
```

Comment:
We should try to use colors in our theme instead of introducing new ones. We have storybook at https://storybook.getsentry.net with a section on colors if you wanted to view them in a browser.